dynawo/sources/Models/CPP/ModelNetwork/DYNModelNetwork.h

```diff
@@ -307,9 +307,9 @@ class ModelNetwork : public ModelCPP::Impl, private boost::noncopyable {
 
   /**
    * @brief scan through the AC network to find AC-connected components
-   *
+   * @param t : time to use
```

Comment:
if I understrand well time is only here for log purposes, maybe add "(only use for log purpose)" in the comment ? same goes for Bus